app/models/hackney/keyfax_session.rb

```diff
@@ -0,0 +1,15 @@
+class Hackney::KeyfaxSession
+  include ActiveModel::Model
+
+  attr_accessor :launch_url, :guid
+
+  def self.create(current_page_url)
```

Comment:
keyword argument `current_page_url:`, so it will accept a hash and conform to the ActiveRecord.

Reply:
:sparkles: